packages/common/src/fixtures/geopoint/geopoint-with-maps.xml

```diff
@@ -126,16 +126,19 @@
       <bind nodeset="/data/maps_group/select_to_disable" type="string" required="true()"/>
       <bind nodeset="/data/maps_group/select_to_show" type="string"/>
       <bind nodeset="/data/maps_group/default_value" type="string"/>
-      <bind nodeset="/data/maps_group/geo_maps" type="geopoint" calculate=" /data/maps_group/default_value "
-            relevant=" /data/maps_group/select_to_show  = 'yes'" readonly=" /data/maps_group/select_to_disable  = 'yes'"
+      <bind nodeset="/data/maps_group/geo_maps" type="geopoint"
+            calculate="if(/data/maps_group/geo_maps = '', /data/maps_group/default_value, /data/maps_group/geo_maps)"
```

Comment:
`coalesce` might work here?

https://getodk.github.io/xforms-spec/#fn:coalesce

Reply:
Oh yeah, you're right, cleaner.